### ISSUE TITLE
[Crash] [2.0.0-beta6] Crash when selecting How To Play to use Mod Manager

### ISSUE THREAD

**Issue by DakotaWorzella** [Raytwo/ARCropolis#99]
### Your setup
System firmware version: ``12.1.0``
Atmosphère version: ``0.19.5``
Smash version: ``12.0.0``
ARCropolis version: ``2.0.0-beta6``
microSD card format: ``FAT32``

### Reproducibility 
This was done with at least 83 mods that I have in my /ultimate/mods/ folder. Each of those having a info.toml created from a .txt file, and a preview.webp file.

### What happened and what's the context?
I was checking the info.toml and preview.webp's that I've created for mods that don't have any originally included with them, to ensure that they were all working fine.

### Configuration
```
[infos]
version = "2.0.0"

[paths]
arc = "rom:/arc"
umm = "sd:/ultimate/mods"
extra_paths = []

[updater]
server_ip = "104.248.50.215"
beta_updates = false

[logger]
logger_level = "Info"

[misc]
debug = false
region = "us_en"
```

### Crash report
![01626967501_01006a800016e000](https://user-images.githubusercontent.com/24654288/126668154-a8f3cabb-302b-4d56-8ae1-997718196f85.jpg)
[01626967501_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/6863524/01626967501_01006a800016e000.log)

**Comment by jam1garner** [comment]
Fixed by 16361682

**Comment by jam1garner** [comment]
For future users, after beta-7 releases you'll get a warning popup to inform you if your info.toml is invalid and tell you where the issue is (line/column) then continue into arcadia after you hit "continue"
